quantum/keymap_extras/keymap_swedish_osx_ansi.h

```diff
@@ -0,0 +1,233 @@
+ */
+// Row 1
+#define SE_LABK KC_GRV  // <
+#define SE_1    KC_1    // 1
+#define SE_2    KC_2    // 2
+#define SE_3    KC_3    // 3
+#define SE_4    KC_4    // 4
+#define SE_5    KC_5    // 5
+#define SE_6    KC_6    // 6
+#define SE_7    KC_7    // 7
+#define SE_8    KC_8    // 8
+#define SE_9    KC_9    // 9
+#define SE_0    KC_0    // 0
+#define SE_PLUS KC_MINS // +
+#define SE_ACUT KC_EQL  // ´ (dead)
+// Row 2
+#define SE_Q    KC_Q    // Q
+#define SE_W    KC_W    // W
+#define SE_E    KC_E    // E
+#define SE_R    KC_R    // R
+#define SE_T    KC_T    // T
+#define SE_Y    KC_Y    // Y
+#define SE_U    KC_U    // U
+#define SE_I    KC_I    // I
+#define SE_O    KC_O    // O
+#define SE_P    KC_P    // P
+#define SE_ARNG KC_LBRC // Å
+#define SE_DIAE KC_RBRC // ¨ (dead)
+#define SE_QUOT KC_NUHS // '
+// Row 3
+#define SE_A    KC_A    // A
+#define SE_S    KC_S    // S
+#define SE_D    KC_D    // D
+#define SE_F    KC_F    // F
+#define SE_G    KC_G    // G
+#define SE_H    KC_H    // H
+#define SE_J    KC_J    // J
+#define SE_K    KC_K    // K
+#define SE_L    KC_L    // L
+#define SE_ODIA KC_SCLN // Ö
+#define SE_ADIA KC_QUOT // Ä
+// Row 4
+#define SE_Z    KC_Z    // Z
+#define SE_X    KC_X    // X
+#define SE_C    KC_C    // C
+#define SE_V    KC_V    // V
+#define SE_B    KC_B    // B
+#define SE_N    KC_N    // N
+#define SE_M    KC_M    // M
+#define SE_COMM KC_COMM // ,
+#define SE_DOT  KC_DOT  // .
+#define SE_MINS KC_SLSH // -
+
+/* Shifted symbols
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬─────┐
+ * │ > │ ! │ " │ # │ € │ % │ & │ / │ ( │ ) │ = │ ? │ ` │     │
+ * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬───┤
+ * │     │   │   │   │   │   │   │   │   │   │   │   │ ^ │ * │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴───┤
+ * │      │   │   │   │   │   │   │   │   │   │   │   │      │
+ * ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴──────┤
+ * │        │   │   │   │   │   │   │   │ ; │ : │ _ │        │
+ * ├─────┬──┴─┬─┴───┼───┴───┴───┴───┴───┴───┼───┴─┬─┴──┬─────┤
+ * │     │    │     │                       │     │    │     │
+ * └─────┴────┴─────┴───────────────────────┴─────┴────┴─────┘
+ */
+// Row 1
+#define SE_RABK S(SE_LABK) // >
+#define SE_EXLM S(SE_1)    // ! 
+#define SE_DQUO S(SE_2)    // "
+#define SE_HASH S(SE_3)    // #
+#define SE_CURR S(SE_4)    // €
+#define SE_PERC S(SE_5)    // %
+#define SE_AMPR S(SE_6)    // &
+#define SE_SLSH S(SE_7)    // /
+#define SE_LPRN S(SE_8)    // (
+#define SE_RPRN S(SE_9)    // )
+#define SE_EQL  S(SE_0)    // =
+#define SE_QUES S(SE_PLUS) // ?
+#define SE_GRV  S(SE_ACUT) // `
+// Row 2
+#define SE_CIRC S(SE_DIAE) // ^ (dead)
+#define SE_ASTR S(SE_QUOT) // *
+// Row 4
+#define SE_SCLN S(SE_COMM) // ;
+#define SE_COLN S(SE_DOT)  // :
+#define SE_UNDS S(SE_MINS) // _
+
+/* Alted symbols
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬─────┐
+ * │ ≤ │ © │ @ │ £ │ $ │ ∞ │   │ | │ [ │ ] │ ≈ │ ± │   │     │
```

Comment:
Alt+1 and Alt+\ on the ANSI variant appear to be swapped:
![image](https://user-images.githubusercontent.com/4781841/123678286-79c4ec80-d889-11eb-8ec2-e26c357ac188.png)

Reply:
> Alt+1 and Alt+\ on the ANSI variant appear to be swapped:

I have fixed the missing §. But I don't understand the swapped part. On the mac layout the @ is on Alt + 1.